.github/assets/supportedVersion

```diff
@@ -1 +1 @@
-1.19.83.01
+1.20.0.0
```

Comment:
```suggestion
1.20.0.01
```
This need to be the exact same number that the GH workflow detected, it would complain otherwise ;-)